### ISSUE TITLE
Exception: missing System.ServiceModel on fresh Windows 10 1809

### ISSUE THREAD

**Issue by Andrzej-W** [gitextensions/gitextensions#6414]
## Current behaviour
Each time I try `GitExt Open repository` from Windows Explorer context menu I get an exception:
> Could not load file or assembly 'System.ServiceModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies. The module was expected to contain an assembly manifest.

Stack trace:
```
   at GitUI.UserControls.RevisionGrid.Graph.RevisionGraphRevision..ctor(ObjectId objectId, Int32 guessScore)
   at GitUI.UserControls.RevisionGrid.Graph.RevisionGraph.Add(GitRevision revision, RevisionNodeFlags types)
   at GitUI.UserControls.RevisionGrid.RevisionDataGridView.Add(GitRevision revision, RevisionNodeFlags types)
   at GitUI.RevisionGridControl.<>c__DisplayClass149_0.<ForceRefreshRevisions>g__CheckUncommittedChanged|8(ObjectId filteredCurrentCheckout)
   at GitUI.RevisionGridControl.<>c__DisplayClass149_0.<ForceRefreshRevisions>g__OnRevisionRead|0(GitRevision revision)
   at System.Reactive.AnonymousSafeObserver`1.OnNext(T value)
   at System.Reactive.ScheduledObserver`1.Dispatch(ICancelable cancel)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
```
## Did this work in previous version of GitExtensions
I have two computers with similar software installed. On the first one GitExtensions works as expected. On the second one I see the exception. The difference between these computers is:
1. Initially I installed Windows 10 x64 1709 (maybe older) with older version of GitExtensions (probably 2.5x). I upgraded this computer two times to Windows 10 1803 and 1809. I also upgraded GitExtensions from 2.5x to 3.0.0 and recently to 3.0.2.
2. Second computer is fresh. I installed Windows 10 x64 1809 and GitExtensions 3.0.2. It has only the newest .NET framework (installed with Windows) without any leftovers from previous releases.

## Environment

- GitExtensions version: 3.0.2.5232
- GIT version: 2.21.0
- OS version: Windows 10 x64 1809 with all updates
- .NET version: 4.7.03190 (Release 461814) - info from Windows registry

**Comment by RussKie** [comment]
> 2\. Second computer is fresh. I installed Windows 10 x64 1809 and GitExtensions 3.0.2. It has only the newest .NET framework (installed with Windows) without any leftovers from previous releases.

To confirm this is the computer that is having troubles?




> Each time I try `GitExt Open repository` from Windows Explorer context menu I get an exception:

Do you get the same exception if you launch the app via the normal shortcut?

**Comment by Andrzej-W** [comment]
I have just made a new Windows installation in virtual machine and everything works as expected. It is not a bug in GitExtenesions. I have compared this file:
`c:\Windows\Microsoft.NET\assembly\GAC_MSIL\System.ServiceModel\v4.0_4.0.0.0__b77a5c561934e089\System.ServiceModel.dll`
on a new installation with file on my computer and noticed that the first 0x47000 bytes are overwritten. It is not a virus but some data which looks like info displayed in device manager (device identifiers, drivers' version numbers, etc.). Maybe it is a result of bug in SSD firmware or Windows. I have restored this file and now everything works as expected.

Closing as not related to GitExtensions.
